File: app/src/main/java/com/kieran/winnipegbusbackend/TripPlanner/classes/TransferSegment.java
package com.kieran.winnipegbusbackend.TripPlanner.classes;

import org.json.JSONObject;

public class TransferSegment extends Segment {
    public TransferSegment(JSONObject segment, String fromTitle, String toTitle) {
        super(segment, fromTitle, toTitle);
    }
}

File: app/src/main/java/com/kieran/winnipegbusbackend/winnipegtransit/TripPlanner/classes/TransferSegment.kt
package com.kieran.winnipegbusbackend.winnipegtransit.TripPlanner.classes

import org.json.JSONObject

class TransferSegment(tripParameters: TripParameters, segment: JSONObject) : Segment(tripParameters, segment) {
    override fun toString(): String {
        return "Wait for transfer at " + to?.title
    }
}

